specification/schemas/Organization.json

```diff
@@ -23,6 +23,37 @@
         "iban": {
           "$ref": "#/components/schemas/Iban"
         },
+        "language": {
+          "type": "enum",
+          "enum": [
+            "english"
+          ],
+          "example": "english"
```

Comment:
@SAMDevelopment @nessy @M4tini @NickVries @denniskoster 
Did we decide on how we wanted to have languages in the API? I think it's better to use ISO codes here instead of the names of the language (https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes). Mostly because we're also using ISO codes for currencies and countries etc.

Reply:
Besides, English is with a capital E.

Reply:
I agree with @Shodske. Maybe it even makes sense to create a separate language resource for future association with countries?

Reply:
I agree with ISO_639-1 two-letter codes. And I think we don't need a language resource + endpoint + relationships, because this will add complexity and the requirement to use our language_id instead of the ISO code.